Dockerfile

```diff
@@ -37,21 +38,31 @@ RUN pip3 install \
     pytest
 
 #  Hack antlr3 as required by opengeode
-RUN mkdir -p tmp ;	cd /tmp ; wget -q -O - https://download.tuxfamily.org/taste/antlr3_python3_runtime_3.4.tar.bz2 | tar jxpvf - ; cd antlr3_python3_runtime_3.4 ; python3 -m pip install --upgrade .
+RUN mkdir -p tmp \
+    && cd /tmp \
```

Comment:
drop tmp usage, so you'll stay in the current folder (happy debugging if next layers will assume some workdir ;) )

Reply:
Removed tmp, but this does not affect workdir